gunicorn/config.py

```diff
@@ -346,6 +346,15 @@ def validate_pos_int(val):
     return val
 
 
+def validate_float(val):
+    if not val:
+        return 0.0
```

Comment:
Default value `0.05` and `* random.uniform(0, 2)` should give the same default expected value.

Old:
`0.1 * 0.5 = 0.05` average
`0.1 * 0 = 0` min
`0.1 * 1 = 0.1` max

New:
`0.05 * 1 = 0.05` average
`0.05 * 0 = 0` min
`0.05 * 2 = 0.1` max

No?